src/red-lentil-soup.md

```diff
@@ -0,0 +1,29 @@
+# Red Lentil Soup
+
+- 🍳Cook time: 10 min
+
+## Ingredients
+
+- Red Lentils
+- Kidney Beans
+- Basil; Bay Leaves; Rosemary; Thyme; Paprika
+- Kosher Salt
+- Spicy Chorizo Sausage
+- Vegetable Oil
+
+## Directions
+
+1. Boil water in pot.
+2. Add spices and salt.
```

Comment:
Elaborate about the measurements for each of these instructions, please.

Reply:
No because as stated at the top, the measurements are up to the Chef. The directions indicate how much of ingredients to add based on how much water you are using, i.e: 1 Tsp oil for every 250ml water.

Reply:
Ok.